<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartEditorDialog</class>
 <widget class="QMainWindow" name="StartEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="scenarios_widget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_scenario_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать сценарий</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_scenario_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить сценарий</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_scenario_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать сценарий</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
